### PR TITLE
Maintenance update 2021-09-09

### PR DESCRIPTION
**Pull request by nuald**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,96 +63,96 @@ Supports two mode:
...
-|                   Python |  227.596<sub>±5.242</sub> |    10.34<sub>±00.04</sub> + 0.00<sub>±00.00</sub> |  5115.22<sub>±223.46</sub> |
-|                 Tcl (FP) |  285.989<sub>±6.642</sub> |     4.25<sub>±00.08</sub> + 0.00<sub>±00.00</sub> |  5522.61<sub>±280.47</sub> |
-|                     Perl |  354.862<sub>±6.481</sub> |     6.51<sub>±00.05</sub> + 0.00<sub>±00.00</sub> |  7796.46<sub>±338.55</sub> |
-|                Tcl (OOP) | 544.022<sub>±11.873</sub> |     4.25<sub>±00.08</sub> + 0.00<sub>±00.00</sub> | 11120.82<sub>±814.03</sub> |
+|                 Language |                  Time, s |                                       Memory, MiB |                  Energy, J |
+| :----------------------- | -----------------------: | ------------------------------------------------: | -------------------------: |
+|  Racket (Syntax Objects) |   1.404<sub>±0.026</sub> |   111.62<sub>±00.16</sub> + 0.00<sub>±00.00</sub> |     32.75<sub>±01.55</sub> |
+|                  C++/g++ |   1.446<sub>±0.015</sub> |     1.51<sub>±00.01</sub> + 0.00<sub>±00.00</sub> |     31.37<sub>±00.58</sub> |
```
Comment (kostya) [inline README.md:69]: interesting why gcc-11 slower

File: README.md [hunk 2]
```diff
@@ -215,116 +215,116 @@ have some inaccuracies in number parsing:
...
-|                      Vala/clang |  5.602<sub>±0.056</sub> |      0.00<sub>±00.00</sub> + 0.00<sub>±00.00</sub> | 101.12<sub>±00.60</sub> |
-|               Perl (JSON::Tiny) | 12.605<sub>±0.279</sub> |  125.30<sub>±00.07</sub> + 528.63<sub>±00.10</sub> | 231.12<sub>±05.85</sub> |
-|        Ruby/truffleruby (--jvm) | 16.962<sub>±0.464</sub> | 721.72<sub>±12.93</sub> + 1648.53<sub>±20.81</sub> | 479.86<sub>±14.66</sub> |
-|                Ruby/truffleruby | 40.369<sub>±0.806</sub> | 738.03<sub>±01.52</sub> + 2269.36<sub>±70.04</sub> | 834.73<sub>±65.49</sub> |
+|                        Language |                 Time, s |                                        Memory, MiB |                Energy, J |
+| :------------------------------ | ----------------------: | -------------------------------------------------: | -----------------------: |
+| C++/g++ (DAW JSON Link NoCheck) |  0.095<sub>±0.004</sub> |    112.69<sub>±00.01</sub> + 0.00<sub>±00.00</sub> |    2.60<sub>±00.21</sub> |
+|    C++/g++ (simdjson On-Demand) |  0.098<sub>±0.002</sub> |   113.27<sub>±00.03</sub> + 59.81<sub>±00.00</sub> |    2.14<sub>±00.05</sub> |
```
Comment (kostya) [inline README.md:221]: this is updated to master libraries, or the same?

File: README.md (hunk 1 above)
Comment (nuald) [reply README.md:69]: I'm not sure why, but I could confirm that it's definitely an issue with the GCC as I have both 10 and 11, and could see the degradation:

```
root@37056ca3ed45:/src/brainfuck# g++-10 -Wall -O3 -march=native -flto -Wa,-mbranches-within-32B-boundaries -std=c++2a -o target/bin_cpp bf.cpp -I../common/libnotify ../common/libnotify/target/libnotify.a
root@37056ca3ed45:/src/brainfuck# QUIET=1 ../xtime.rb target/bin_cpp bench.b
"C++/g++"
Output checksum: 23280
0.868 s, 1 Mb, 16.2 J

root@37056ca3ed45:/src/brainfuck# g++-11 -Wall -O3 -march=native -flto -Wa,-mbranches-within-32B-boundaries -std=c++2a -o target/bin_cpp bf.cpp -I../common/libnotify ../common/libnotify/target/libnotify.a
root@37056ca3ed45:/src/brainfuck# QUIET=1 ../xtime.rb target/bin_cpp bench.b
"C++/g++"
Output checksum: 23280
1.505 s, 1 Mb, 28.4 J
```
0.868s with g++10 vs 1.505s with g++11

File: README.md (hunk 2 above)
Comment (nuald) [reply README.md:221]: It's always using the latest master branch as I do the full clean before maintenance updates.

**Pull request merged**